### PR TITLE
navigation_ctrl converted to es6 

### PR DESCRIPTION
**Pull request by karinashenaz**

### CODE REVIEW

File: assets/js/controllers/navigation_ctrl.js.coffee [hunk 1]
```diff
@@ -1,42 +0,0 @@
...
-  #################################
-  
-
-  $scope.doLogout = () ->   
-    $translate("ARE_YOU_SURE_LOGOUT").then (translation) ->      
-      if confirm translation
-        $scope.uid = null
-        $scope.password = null
-        $cookieStore.remove("password")
-        # $cookieStore.remove("uid") // Pending a "Forget Me feature"
```
Comment (Sjors) [inline assets/js/controllers/navigation_ctrl.js.coffee]: @karinashenaz this comment was lost in the conversion.

File: assets/js/controllers/navigation_ctrl.js.coffee [hunk 2]
```diff
@@ -1,42 +0,0 @@
...
-        $state.go("wallet.common.transactions", {accountIndex: "accounts"})
-        Wallet.logout() # Refreshes the browser, so won't return
```
Comment (Sjors) [inline assets/js/controllers/navigation_ctrl.js.coffee]: comment

File: assets/js/controllers/navigation_ctrl.js.coffee (hunk 1 above)
Comment (karinashenaz) [inline assets/js/controllers/navigation_ctrl.js.coffee]: put back

File: assets/js/controllers/navigation_ctrl.js.coffee (hunk 2 above)
Comment (karinashenaz) [inline assets/js/controllers/navigation_ctrl.js.coffee]: put back now

**Pull request merged**